specs/windows/groups.table

```diff
@@ -0,0 +1,17 @@
+table_name("groups")

```

Comment:
A "duplicate" specfile will cause an error. Specfiles are global, but hidden on systems for which they do not apply. Can you move the existing `groups.table` into `./specs/` and then apply the `extended_schema` to that?

Reply:
Done